### PR TITLE
[#156] Update docs

### PR DESCRIPTION
**Pull request by rvem**
## Description
Problem: Now we're providing more user-friendly way to set up and
start baking instace, we should use this way in the baking doc.

Solution: Rework baking.md to use `tezos-baking-<network>` service.

~~Currently based on #164.~~ Don't merge until updated packages are published.

## Related issue(s)

Relates #156

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

- [x] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: docs/baking.md [hunk 1]
```diff
@@ -101,6 +106,13 @@ In order to import the snapshot, run the following command:
 sudo -u tezos tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
 ```
 
+Note that even after the snapshot import the node can still be out of sync. It may require
```
Comment (rvem) [inline docs/baking.md:109]: This part should probably be moved to the `Starting baking instance` section

**Review by pasqu4le — COMMENTED** [review]
It seems some reordering is necessary.
Aside from that I checked the guide on both Ubuntu and Raspberry Pi and all seems to work smoothly.

File: docs/baking.md [hunk 2]
```diff
@@ -177,71 +142,97 @@ the key:
 
 In order to activate account run:
 ```
-sudo -u tezos tezos-client activate account <alias> <path-to-downloaded-json>
+sudo -u tezos tezos-client activate account baker <path-to-downloaded-json>
```
[the remark this answers is not in the document]
Comment (rvem) [reply docs/baking.md:145]: :facepalm:

File: docs/baking.md [hunk 3]
```diff
@@ -123,50 +128,10 @@ sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --
...
-By default, both these daemons have `/var/lib/tezos/.tezos-client` set as a `DATA_DIR`.
-
-Additionally, you need to specify `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>`
-to point at the desired node data directory, e.g. `/var/lib/tezos/node-<network>`.
+## Setting up baker and endorser key
 
+By default `tezos-baking-<network>.service` will use `baker` alias for the
+key that will be used for baking and endorsing.
 
 ### Importing the baker key
```
[the remark this answers is not in the document]
Comment (rvem) [reply docs/baking.md:136]: Also, I think we can move `Registering the baker` section to the very end after `Starting baking instance`

[the remark this answers is not in the document]
Comment (rvem) [reply docs/baking.md:136]: > There's a problem here now, as the node hasn't started yet the tezos-client commands below will fail.

Hmm, indeed, though, only `tezos-client activate account` will fail in case there is no node, the rest of imports' commands will just show a warning

File: docs/baking.md [hunk 4]
```diff
@@ -20,29 +20,36 @@ and an arm64 processor; for example a Raspberry Pi 4B.
...
 `tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
 
-The currently used proto is `008-PtEdo2Zk` (used on `mainnet` and `edo2net`).
+The currently supported protos are `008-PtEdo2Zk` (used on `mainnet` and `edo2net`)
+and `009-PsFLoren` (used on `florencenet`).
 Also, note that the corresponding packages have protocol
 suffix in lowercase, e.g. the list of available baker packages can be found
 [here](https://launchpad.net/~serokell/+archive/ubuntu/tezos/+packages?field.name_filter=tezos-baker&field.status_filter=published).
 
+The most convenient way to orchestrate all these binaries is to use `tezos-baking`
```
Comment (pasqu4le) [inline docs/baking.md:36]:
```suggestion
The most convenient way to orchestrate all these binaries is to use the `tezos-baking`
```

File: docs/baking.md [hunk 5]
```diff
@@ -52,35 +59,33 @@ sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bion
...
-systemd units for running the corresponding binaries in the background.
-
-## Setting up the tezos-node
-
-The easiest way to set up a node running on the `mainnet` or one of the
-testnets (e.g. `edo2net`) is to use one of the predefined
-`tezos-node-<network>.services` systemd services provided in the `tezos-node`
-package.
+systemd units for running the corresponding binaries in the background, these units
+are orchestrated by the services provided by the `tezos-baking` package.
```
Comment (pasqu4le) [inline docs/baking.md:67]: seems kinda repetitive mentioning the `tezos-baking` package here, maybe instead say:
```suggestion
are orchestrated by the `tezos-baking-<network>` units.
```

File: docs/baking.md [hunk 6]
```diff
@@ -52,35 +59,33 @@ sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bion
...
 
-However, by default, these services will start to bootstrap the node from scratch,
-which will take a significant amount of time.
-In order to avoid this, we suggest bootstrapping from a snapshot instead.
+## Setting up baking service
 
-### Setting up node service
+`tezos-baking-<network>.service` uses `/var/lib/.tezos-client` as a default
+`tezos-client` data directory and `/var/lib/tezos/node-<network>` as `tezos-node`
+data directory. By default node is going to use `http://localhost:8732` as RPC address.
```
Comment (pasqu4le) [inline docs/baking.md:73]: Maybe make it a bullet list?

File: docs/baking.md [hunk 7]
```diff
@@ -52,35 +59,33 @@ sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bion
...
 
-`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
-and `http://localhost:8732` as its RPC address by default.
+In case you want to use a different `tezos-client` data directory or RPC address,
+you should update `/etc/default/tezos-baking-<network>` file.
```
Comment (pasqu4le) [inline docs/baking.md:76]: Can `sudo systemctl edit` be used here as well?

File: docs/baking.md [hunk 8]
```diff
@@ -177,71 +152,97 @@ the key:
...
 Check a blockchain explorer (e.g. https://tzkt.io/ or https://tzstats.com/) to see the baker status and
 baking rights of your account.
 
-### Updating baker and endorser daemons configuration
-Apart from that, you'll need to update the `BAKER_ACCOUNT` and `ENDORSER_ACCOUNT` (in
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>` respectively) in
-accordance to the **alias** of the imported key.
+### Using different baking account alias
+
+By default `tezos-baking-<network>.service` will use `baker` alias for the
```
Comment (pasqu4le) [inline docs/baking.md:189]:
```suggestion
By default `tezos-baking-<network>.service` will use the `baker` alias for the
```

File: docs/baking.md (hunk 7 above)
Comment (rvem) [reply docs/baking.md:76]: hm, no, `sudo systemctl edit` is supposed to be used  only for editing systemd files

Comment (pasqu4le) [reply docs/baking.md]: Oh yeah :facepalm: got confused

File: docs/baking.md (hunk 6 above)
Comment (pasqu4le) [reply docs/baking.md:73]: Oh I see you made the changing instructions that are following in a bullet list, but I meant the ones above.
```
By default `tezos-baking-<network>.service` will be using:
- `/var/lib/.tezos-client` as the `tezos-client` data directory
- `/var/lib/tezos/node-<network>` as the `tezos-node` data directory
- `http://localhost:8732` as the `tezos-node` RPC address.
```
what do you think?

File: docs/baking.md [hunk 9]
```diff
@@ -159,71 +137,97 @@ the key:
...
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>` respectively) in
-accordance to the **alias** of the imported key.
+### Using different baking account alias
+
+By default `tezos-baking-<network>.service` will use the `baker` alias for the
+key that will be used for baking and endorsing.
+
+In case you want to use different alias for the baking account, follow two previous sections
+and replace `baker` with the desired alias. Also, you'll need to update `BAKER_ADDRESS_ALIAS`
+in `/etc/default/tezos-baking-<network>`.
```
Comment (pasqu4le) [inline docs/baking.md:179]: :thinking: I though I made this comment before, but maybe it kinda got lost between the other changes.

I think this should be right before the `Importing the baker key` section above.

IMO if one want to change the alias he'll want to do so before reading the instructions to import the key.

Comment (rvem) [reply docs/baking.md:179]: makes sense

File: docs/baking.md (hunk 6 above)
Comment (rvem) [reply docs/baking.md:73]: hmm, yeah, that looks good

File: docs/baking.md [hunk 10]
```diff
@@ -20,29 +20,36 @@ and an arm64 processor; for example a Raspberry Pi 4B.
...
-with an image downloasd from the [official 64bit repository](https://downloads.raspberrypi.org/raspios_arm64/images/).
+with an image downloaded from the [official 64bit repository](https://downloads.raspberrypi.org/raspios_arm64/images/).
 
 ### Installing required packages
 
 In order to run a baking instance, you'll need the following Tezos binaries:
 `tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
 
-The currently used proto is `008-PtEdo2Zk` (used on `mainnet` and `edo2net`).
+The currently supported protos are `008-PtEdo2Zk` (used on `mainnet` and `edo2net`)
```
Comment (itkach) [inline docs/baking.md:30]: I think it's best to not abbreviate here... protos -> protocols

File: docs/baking.md [hunk 11]
```diff
@@ -105,50 +114,27 @@ In order to import the snapshot, run the following command:
...
+running fully-bootstrapped `tezos-node`. In order to start node service do the following:
 ```
 sudo systemctl start tezos-node-<network>.service
 ```
 
-Note that even after the snapshot import the node can still be out of sync. It may require
+Even after the snapshot import the node can still be out of sync. It may require
 some additional time to completely bootstrap. In order to check whether the node is bootstrapped,
-you can use `tezos-client`:
+you can use tezos-client:
```
Comment (itkach) [inline docs/baking.md:127]: to check and wait if it's not bootsrapped

File: docs/baking.md [hunk 12]
```diff
@@ -105,50 +114,27 @@ In order to import the snapshot, run the following command:
...
-Data directories for baker and endorser daemons are defined in the
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>`.
-By default, both these daemons have `/var/lib/tezos/.tezos-client` set as a `DATA_DIR`.
-
-Additionally, you need to specify `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>`
-to point at the desired node data directory, e.g. `/var/lib/tezos/node-<network>`.
+By default `tezos-baking-<network>.service` will use the `baker` alias for the
+key that will be used for baking and endorsing.
 
+In case you want to use different alias for the baking account, replace `baker` with
```
Comment (itkach) [inline docs/baking.md:135]: I would move instructions for customizations like  to the "Advanced" section at the very end of the document. There is no reason for users to think about this, know about this or change this, ever. Of course there will be users who want to do this anyway, so... Advanced section :)

File: docs/baking.md [hunk 13]
```diff
@@ -52,35 +59,37 @@ sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bion
...
 
-### Setting up node service
+By default `tezos-baking-<network>.service` will be using:
+* `/var/lib/.tezos-client` as the `tezos-client` data directory
+* `/var/lib/tezos/node-<network>` as the `tezos-node` data directory
+* `http://localhost:8732` as the `tezos-node` RPC address.
 
-`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
-and `http://localhost:8732` as its RPC address by default.
+In case you want to use a different `tezos-client` data directory or RPC address,
```
Comment (itkach) [inline docs/baking.md:76]: Similar to customizing baker key alias, I think this should go to the end of the document, into "Advanced" section (which should probably also elaborate on why users may want to change this - e.g. not having adequate storage in /var). Moving it to the end will get majority of users to a working setup quicker.

File: docs/baking.md [hunk 14]
```diff
@@ -105,50 +114,27 @@ In order to import the snapshot, run the following command:
 sudo -u tezos tezos-node-<network> snapshot import <path to the snapshot file>
 ```
 
-### Starting the node
+## Setting up baker and endorser key
 
-After the snapshot import, you can finally start the node by running:
+Note that account activation from JSON file and baker registering require
```
Comment (itkach) [inline docs/baking.md:119]: I would prioritize (that is - mention first) information that is relevant for mainnet: 1) import key from ledger nano s 2) import secret key directly 3) generate new key pair, then fund + delegate 4) use remote signer, with details for 3 and 4 moved to "Advanced" section at the end.

File: docs/baking.md [hunk 15]
```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
...
+running fully-bootstrapped `tezos-node`. In order to start node service do the following:
```
Comment (pasqu4le) [inline docs/baking.md:107]:
```suggestion
running a fully-bootstrapped `tezos-node`. In order to start node service do the following:
```

File: docs/baking.md [hunk 16]
```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
...
 ```
 sudo systemctl start tezos-node-<network>.service
 ```
 
-Note that even after the snapshot import the node can still be out of sync. It may require
-some additional time to completely bootstrap. In order to check whether the node is bootstrapped,
-you can use `tezos-client`:
+Even after the snapshot import the node can still be out of sync. It may require
+some additional time to completely bootstrap. In order to check whether the node
+is bootstrapped and wait in case it isn't, you can use `tezos-client`:
```
Comment (pasqu4le) [inline docs/baking.md:114]:
```suggestion
Even after the snapshot import the node can still be out of sync and may require
some additional time to completely bootstrap.

In order to check whether the node is bootstrapped and wait in case it isn't,
you can use `tezos-client`:
```
I think would be a bit more readable in the rendered doc.

File: docs/baking.md [hunk 17]
```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
...
 baking rights of your account.
 
-### Updating baker and endorser daemons configuration
-Apart from that, you'll need to update the `BAKER_ACCOUNT` and `ENDORSER_ACCOUNT` (in
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>` respectively) in
-accordance to the **alias** of the imported key.
+## Starting baking instance
 
-### Starting daemons
+Once the key is imported and the configs are updated (if needed), you can start your baking instance:
```
Comment (pasqu4le) [inline docs/baking.md:168]: There is no previous mention of config update before this anymore, so this is quite confusing to read, I suggest:
```suggestion
Once the key is imported and the baker registered, you can start your baking instance:
```

File: docs/baking.md [hunk 18]
```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
...
+sudo systemctl enable tezos-baking-edo2net
 ```
 
-Logs for successfully started endorser service should begin with:
+## Advanced baking instance setup
+
+### Using different data directories and node RPC address
+
+In case you want to use a different `tezos-client` data directory or RPC address,
+you should update `/etc/default/tezos-baking-<network>` file, e.g.:
```
Comment (pasqu4le) [inline docs/baking.md:225]:
```suggestion
you should edit the `/etc/default/tezos-baking-<network>` file, e.g.:
```

File: docs/baking.md [hunk 19]
```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
...
 ```
-Node is bootstrapped.
-Endorser started.
+sudo vim /etc/default/tezos-baking-<network>
+```
+
+In case you want to use a different `tezos-node` data directory, you
+should edit the service configuration. To edit the service configuration, run:
```
Comment (pasqu4le) [inline docs/baking.md:231]:
```suggestion
In case you want to use a different `tezos-node` data directory, you
should instead edit the service configuration, using:
```

File: docs/baking.md [hunk 20]
```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
...
 ```
+sudo systemctl edit --full tezos-node-<network>.service
+```
+
+### Using different account alias for baking
+
+In case you want to use different alias for the baking account, replace `baker` with
+the desired alias in sections about [importing](#import) and [registering](#registration)
+baker key. Also, you'll need to update `BAKER_ADDRESS_ALIAS` in
+`/etc/default/tezos-baking-<network>`.
```
Comment (pasqu4le) [inline docs/baking.md:241]: The `BAKER_ADDRESS_ALIAS` seems to be left as an after-thought, I suggest:
```suggestion
In case you want to use a different alias for the baking account:
1. replace `baker` with he desired alias in the sections about [importing](#import)
    and [registering](#registration) the baker key.
2. update the `BAKER_ADDRESS_ALIAS` by editing the
    `/etc/default/tezos-baking-<network>` file.
```

File: docs/baking.md [hunk 21]
```diff
@@ -105,125 +101,144 @@ In order to import the snapshot, run the following command:
 sudo -u tezos tezos-node-<network> snapshot import <path to the snapshot file>
 ```
 
-### Starting the node
+## Setting up baker and endorser key
 
-After the snapshot import, you can finally start the node by running:
+Note that account activation from JSON file and baker registering require
+running a fully-bootstrapped `tezos-node`. In order to start node service do the following:
```
Comment (pasqu4le) [inline docs/baking.md:107]:
```suggestion
running a fully-bootstrapped `tezos-node`. In order to start the node service do the following:
```

File: docs/baking.md [hunk 22]
```diff
@@ -105,125 +101,144 @@ In order to import the snapshot, run the following command:
...
-3) The secret key is stored on a ledger.
 
-Open the Tezoz Wallet app on your ledger and run the following
-to import the key:
-```
-sudo -u tezos tezos-client import secret key <alias> <ledger-url>
-```
-Apart from importing the key, you'll also need to set it up for baking. Open Tezos Baking app
-on your ledger and run the following:
+3) You have faucet JSON file from https://faucet.tzalpha.net/.
```
Comment (pasqu4le) [inline docs/baking.md:150]:
```suggestion
3) You have a faucet JSON file from https://faucet.tzalpha.net/.
```

File: docs/baking.md [hunk 23]
```diff
@@ -105,125 +101,144 @@ In order to import the snapshot, run the following command:
...
+
+In order to activate account run:
```
Comment (pasqu4le) [inline docs/baking.md:152]:
```suggestion
In order to activate the account run:
```

File: docs/baking.md [hunk 24]
```diff
@@ -105,125 +101,144 @@ In order to import the snapshot, run the following command:
...
+* `tezos-baker-<proto>@<network>.service`
+* `tezos-endorser-<proto>@network.service`
+
+Once services have started, you can check their logs via `journalctl`:
 ```
-sudo systemctl start tezos-baker-<proto>.service
-sudo systemctl start tezos-endorser-<proto>.service
+journalctl -f _UID=$(id tezos -u)
 ```
+This command will show logs for all services that are using `tezos` user.
```
Comment (pasqu4le) [inline docs/baking.md:184]:
```suggestion
This command will show logs for all services that are using the `tezos` user.
```

File: docs/baking.md [hunk 25]
```diff
@@ -105,125 +101,144 @@ In order to import the snapshot, run the following command:
...
 ```
-journalctl -u tezos-baker-<proto>.service
-journalctl -u tezos-endorser-<proto>.service
+sudo systemctl stop tezos-baking-<network>.service
 ```
 
-If everything was set up correctly, you shouldn't see any errors in the logs.
+## Running baking instance on edo2net
+
+TL;DR, in order to run baking instance on edo2net you should do the following:
```
Comment (pasqu4le) [inline docs/baking.md:201]:
```suggestion
TL;DR, in order to run a baking instance on edo2net you should do the following:
```

**Review by pasqu4le — APPROVED** [review]
Minor suggestions, but LGTM now :+1:
